app/src/main/java/ch/epfl/sweng/erpa/model/Game.java

```diff
@@ -0,0 +1,111 @@
+package ch.epfl.sweng.erpa.model;
+
+import java.util.List;
+import java.util.Objects;
+
+public class Game {
+    private String gmUniqueID;
+    private GameParticipants players;
+    private String name;
+    private String minPlayer;
+    private String maxPayer;
+    private String difficulty;
+    private String universe;
+    private String oneshotOrCampaign;
+    private String numberSessions;
+    private String sessionLength;
+    private String description;
+
+    public Game(String gmUniqueID, String name, String minPlayer,
```

Comment:
I'd rather we keep conceptually inmutable classes.
Here's a nice discussion about the subject: https://www.quora.com/Is-it-common-to-make-a-DTO-object-as-an-immutable-object

Reply:
Yeah, Game should stay immutable, but I think it might be a good idea to simplify this constructor. Think of String and StringBuilder. Having a constructor that takes 10 unnamed arguments is bound to lead to confusion and errors.

Reply:
That's an interesting idea, but I believe that all arguments are required, thus there's not much point on making a builder.